packages/legend-studio/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_MappingTransformer.ts

```diff
@@ -442,6 +445,7 @@ const transformRelationalPropertyMapping = (
   element: RelationalPropertyMapping,
   isTransformingEmbeddedPropertyMapping: boolean,
   context: V1_GraphTransformerContext,
+  renderSourceId: boolean,
```

Comment:
renderSourceId? you are not really rendering here. this is used to transfrom to protocol